packages/app/src/app/pages/common/Modals/index.tsx

```diff
@@ -26,7 +26,6 @@ import { LiveVersionMismatch } from './LiveSessionVersionMismatch';
 import { NetlifyLogs } from './NetlifyLogs';
 import { PickSandboxModal } from './PickSandboxModal';
 import { PreferencesModal } from './PreferencesModal';
-import { RecoverFilesModal } from './RecoverFilesModal';
```

Comment:
I think I got everything relating to this modal but would be good to double check